avaframe/com1DFAPy/com1DFA.py

```diff
@@ -1,195 +1,1158 @@
+        xpart = csz * (np.random.rand(nPart) - 0.5 + indx)
+        ypart = csz * (np.random.rand(nPart) - 0.5 + indy)
+    else:
+        # place particles equaly distributed
+        xpart = csz * (- 0.5 + indx) + x
+        ypart = csz * (- 0.5 + indy) + y
+    return xpart, ypart, mPart, nPart
+
+
+def intializeMassEnt(dem):
+    # read dem header
+    header = dem['header']
+    ncols = header.ncols
+    nrows = header.nrows
+    Ment = np.zeros((nrows, ncols))
+    return Ment
+
+
+def intializeResistance(dem):
+    # read dem header
+    header = dem['header']
+    ncols = header.ncols
+    nrows = header.nrows
+    Cres = np.zeros((nrows, ncols))
+    return Cres
+
+
+def DFAIterate(cfg, particles, fields, dem, Ment, Cres, Tcpu):
+    """ Perform computations and save results for desired interval """
+
+    # Load configuration settings
+    Tend = cfg.getfloat('Tend')
+    dtSave = cfg.getfloat('dtSave')
+
+    # Initialise Lists to save fields
+    Particles = [copy.deepcopy(particles)]
+    Fields = [copy.deepcopy(fields)]
+    # save Z, S, U, T at each time step for developping purpouses
+    Z = np.empty((0, 0))
+    S = np.empty((0, 0))
+    U = np.empty((0, 0))
+    T = np.empty((0, 0))
+
+    # Initialize time and counters
+    nSave = 0
+    nIter = 0
+    iterate = True
+    particles['iterate'] = iterate
+    t = particles['t']
+    # Start time step computation
+    while t < Tend and iterate:
+
+        #++++++++++++++++if you want to use cfl time step+++++++++++++++++++
+        # CALL TIME STEP:
+        # to play around with the courant number
+        if featCFL:
+            dtStable = tD.getcflTimeStep(particles, dem, cfg)
+        elif featCFLConstrain:
+            dtStable = tD.getcfldTwithConstraints(particles, dem, cfg)
+
+        # dt overwrites dt in .ini file, so comment this block if you dont want to use cfl
+        #++++++++++++++++++++++++++++++++++++++++++++++
+        # get time step
+        dt = cfg.getfloat('dt')
+        t = t + dt
+        nIter = nIter + 1
+        log.info('Computing time step t = %f s', t)
+        T = np.append(T, t)
+        particles['t'] = t
+        Tcpu['nSave'] = nSave
+
+        # Perform computations
+        if featLF:
+            particles, fields, Tcpu, dt = computeLeapFrogTimeStep(cfg, particles, fields, dt, dem, Ment, Cres, Tcpu)
+        else:
+            particles, fields, Tcpu = computeTimeStep(cfg, particles, fields, dt, dem, Ment, Cres, Tcpu)
+        # Save desired parameters and export to Lists for saving interval
+        U = np.append(U, DFAtls.norm(particles['ux'][0], particles['uy'][0], particles['uz'][0]))
+        Z = np.append(Z, particles['z'][0])
+        S = np.append(S, particles['s'][0])
+        iterate = particles['iterate']
+        if t >= nSave * dtSave:
+            log.info('Saving results for time step t = %f s', t)
+            log.info('MTot = %f kg, %s particles' % (particles['mTot'], particles['Npart']))
```

Comment:
could this be debug level?

Reply:
maybe I leave this one as info for now, I switched the 2 previous prints as debug